bioimageio/core/build_spec/build_model.py

```diff
@@ -477,6 +705,17 @@ def build_model(
         if tmp_source is not None:
             os.remove(tmp_source)
 
+    # breakpoint()
+    # debugging: for some reason this causes a validation error for the sample_input / output,
+    # although they are passed in the same format as test inputs / outputs:
+    # (Pdb) model.sample_inputs
+    # [PosixPath('/tmp/bioimageio_cache/extracted_packages/efae734f36ff8634977c9174f01c854c4e2cfc799bc92751a4558a0edd963da6/sample_input_0.tif')]
+    # (Pdb) model.sample_outputs
+    # [PosixPath('/tmp/bioimageio_cache/extracted_packages/efae734f36ff8634977c9174f01c854c4e2cfc799bc92751a4558a0edd963da6/sample_output_0.tif')]
+    # (Pdb) model.test_inputs
+    # [PosixPath('/tmp/bioimageio_cache/extracted_packages/efae734f36ff8634977c9174f01c854c4e2cfc799bc92751a4558a0edd963da6/test_input.npy')]
+    # (Pdb) model.test_outputs
+    # [PosixPath('/tmp/bioimageio_cache/extracted_packages/efae734f36ff8634977c9174f01c854c4e2cfc799bc92751a4558a0edd963da6/test_output.npy')]
```

Comment:
should be fixed by #164 ; then this should be removed

Reply:
> the tests for writing the deepimagej config is still failing with

... or not.. looking into it

Reply:
the problem is that sample inputs/outputs are absolute and not relative paths...

Reply:
but test inputs / outptus are also absolute paths. Why does it work for them and not for sample inputs/outputs?